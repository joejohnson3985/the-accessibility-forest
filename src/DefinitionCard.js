import React from 'react';
import './DefinitionCard.scss';


const DefinitionCard = (props) => {
  return (
    <div className='definition-container'>
      <h2>Definition</h2>
      <h3>{props.termType}</h3>
      <p>{props.definition}</p>
    </div>
  )
}

export default DefinitionCard;