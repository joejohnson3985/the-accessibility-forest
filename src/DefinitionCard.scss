@import './_variables.scss';

.definition-container {
  width: 60%;
  margin: 5px;
  color: #fff;
}