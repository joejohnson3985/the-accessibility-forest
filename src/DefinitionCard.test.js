import React from 'react';
import DefinitionCard from './DefinitionCard';
import { shallow } from 'enzyme';

const currentDeff = 'Indicates that the enclosed HTML provides contact information for a person or people, or for an organization.'
const currentType = 'html tag'

describe('DefinitionCard', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
        <DefinitionCard 
          definition={currentDeff} 
          termType={currentType}
        />
      )
  });

  it('Should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})