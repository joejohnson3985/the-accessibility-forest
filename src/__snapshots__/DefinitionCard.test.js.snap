// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefinitionCard Should match the snapshot 1`] = `
<div
  className="definition-container"
>
  <h2>
    Definition
  </h2>
  <h3>
    html tag
  </h3>
  <p>
    Indicates that the enclosed HTML provides contact information for a person or people, or for an organization.
  </p>
</div>
`;
